fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerBuildIndexTest.java

```diff
@@ -227,27 +231,18 @@ void singleRebuild(@Nonnull List<TestRecords1Proto.MySimpleRecord> records,
                 } else {
                     // Safe builds do not support building ranges yet.
```

Comment:
Am I correct in my understanding that safe builds use a synchronized session, and non-safe builds do not?
@alecgrieser are you familiar with the goals of this test?

There's probably still an argument for keeping the old code until it is deleted, but it does look like we need additional coverage for mutual indexing here.

Reply:
If mutual indexing needs more coverage tests, I would much rather add them elsewhere.